Aggregator.ConsoleApp/Aggregator.ConsoleApp.csproj

```diff
@@ -1,74 +1,69 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="..\packages\System.Runtime.InteropServices.Analyzers.1.0.1\build\System.Runtime.InteropServices.Analyzers.props" Condition="Exists('..\packages\System.Runtime.InteropServices.Analyzers.1.0.1\build\System.Runtime.InteropServices.Analyzers.props')" />
-  <Import Project="..\packages\System.Runtime.Analyzers.1.0.1\build\System.Runtime.Analyzers.props" Condition="Exists('..\packages\System.Runtime.Analyzers.1.0.1\build\System.Runtime.Analyzers.props')" />
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug-2018</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <ProjectGuid>{200CF240-476E-4429-A8FC-66001806A2DE}</ProjectGuid>
     <OutputType>Exe</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Aggregator.ConsoleApp</RootNamespace>
     <AssemblyName>TFSAggregator2.ConsoleApp</AssemblyName>
-    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
+    <TargetFramework Condition="'$(TargetFramework)'==''">net461</TargetFramework>
     <FileAlignment>512</FileAlignment>
-    <NuGetPackageImportStamp>
-    </NuGetPackageImportStamp>
-    <TargetFrameworkProfile />
+    <Configurations>Debug-2013;Release-2013;Debug-2015;Release-2015;Debug-2015.1;Release-2015.1;Debug-2015.2;Release-2015.2;Debug-2017;Release-2017;Debug-2017.2;Release-2017.2;Debug-2018;Release-2018</Configurations>
```

Comment:
same as I posted in core:

 Ideally, there shouldn't be any Debug-XYZ / Release-XYZ configurations left if all SDK projects inherit a default TfsVersion parameter from a Directory.Build.props file (auto-imported). The properties could then be conditioned on the TfsVersion parameter instead of the configuration (or a set in the Directory.Build.props file).

This would allow adding/removing tfs versions without fiddling with solution and projects configurations.

As long as a restore is performed for each tfsversion (`/restore` argument to msbuild 15.5+) this should work from build scripts. and in vs CPS tracks changes to imported files now (was a 15.3 feature) and should trigger incremental restore in VS. This is the main reason I set the intermediate output path to include the tfs version, so that restore output (project.assets.json) is seggregated by TfsVersion and shouldn't cause conflicts when switching between tfsversion builds.

When set globally, the tfsversion is forwarded to all project references. Alternatively `AdditionalMetadata="TfsVersion=$(...)"` could be set on them if needed.